homeassistant/components/spotify/browse_media.py

```diff
@@ -384,7 +411,7 @@ def _browsing_get_items(media_content_type, spotify):
     media: dict[str, Any] | None = None
 
     if media_content_type == BrowsableMedia.CURRENT_USER_PLAYLISTS:
-        if media := spotify.current_user_playlists(limit=BROWSE_LIMIT):
+        if media := spotify.current_user_top_artists(limit=BROWSE_LIMIT):
```

Comment:
Everything looks nice! I just wonder why do you check if media_content_type is CURRENT_USER_PLAYLIST and then in the second if statement you check the current_user_top_artists instead of current_user_playlists (line: 414)? Sonarcloud is classifying this as a code smell because there is an identical if statement on line: 417.

Reply:
fixed